src/platform/audio/milesaudiofilecache.cpp

```diff
@@ -227,12 +227,12 @@ bool MilesAudioFileCache::Free_Space_For_Sample(const OpenAudioFile &file)
 void MilesAudioFileCache::Release_Open_Audio(OpenAudioFile *file)
 {
     // Close any playing samples that use this data.
-    if (file->ref_count) {
-        g_theAudio->Close_Any_Sample_Using_File(file);
+    if (file->ref_count != 0) {
+        g_theAudio->Close_Any_Sample_Using_File(file->wave_data);
```

Comment:
Thyme bugfix here. I put breakpoint to see if it is called in game, but did not hit, so maybe was not major problem on runtime.

Reply:
Shouldn't you add the `// #BUGFIX` thing here?

Reply:
No, because this is what the original does. It was just wrong in Thyme.

Reply:
Okay, good!